src/style.css

```diff
@@ -0,0 +1,267 @@
+  flex: 1;
+  padding: 0 1em;
+  color: #fff;
+  background-color: #0365ac;
+  background-image: none;
+  cursor: pointer;
+  font-size: 16px;
+}
+
+.select {
+  position: relative;
+  display: flex;
+  width: 20em;
+  height: 3em;
+  border-radius: 0.25em;
+  overflow: hidden;
+  margin-right: 20px;
+}
+
+.select::after {
+  content: "\25BC";
+  position: absolute;
+  top: 0;
+  right: 0;
+  padding: 1em;
+  background-color: #024e85;
+  transition: 0.25s all ease;
+  pointer-events: none;
+}
+
+/* Transition */
+.select:hover::after {
+  color: #29a3fb;
+}
+
+.banner {
+  width: 100%;
+  height: 65vh;
+  background-image: url("https://www.themealdb.com/images/category/chicken.png");
+  background-repeat: no-repeat;
+  background-position: center;
+  display: flex;
+  align-items: flex-end;
+  justify-content: center;
+  margin-bottom: 230px;
+}
+
+.banner h1 {
+  font-size: 45px;
+  color: #dbdbdb;
+  font-family: "Countryside", sans-serif;
+  height: 30%;
+  margin: 0;
+}
+
+.banner span {
+  height: 19%;
+  margin-top: 60px;
+  color: var(--secondary-color);
+}
+
+.innerAligment {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  max-width: 550px;
+  min-width: 250px;
+  margin: 30px;
+  text-align: center;
+  margin-bottom: -70px;
+  height: 140px;
+}
+
+.cards {
+  width: 100%;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: row;
+  grid-gap: 1rem;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
+.flex-card {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  max-width: 100%;
+  transition-property: border-color, box-shadow;
+  transition-duration: 1.5s;
+}
+
+.align_images {
+  margin: auto;
+}
+
+.imagen_proyectos {
+  width: 100%;
+  border-radius: 8px;
+}
+
+.text_child h2 {
+  font-weight: 700;
+  line-height: 25px;
+  line-height: 35px;
+  margin: 0;
+  margin-top: 15px;
+}
+
+.text_child p {
+  color: #344563;
+  word-break: normal;
+
+  /* max-width: fit-content; */
+  max-width: 430px;
+}
+
+.text_child {
+  margin-left: 2%;
+}
+
+.flex_child {
+  display: flex;
+  flex-direction: row;
+}
+
+.card {
+  background-color: #fff;
+  padding: 1rem;
+  margin: 0 2% 0 2%;
+  margin-left: 4%;
+  margin-right: 4%;
+  width: 300px;
+  border: 2px solid #dfe1e6;
+  border-radius: 24px;
+}
+
+.gray_text {
+  color: gray;
+  margin-left: 13px;
+  margin-top: 2.9px;
+  font-size: 0.8rem;
+}
+
+.SeeCommentsButton {
+  border: 1px solid var(--primary-color);
+  border-radius: 8px;
+  box-sizing: border-box;
+  background: #fff;
+  height: 48px;
+  margin-top: 25px;
+  margin-bottom: 19px;
+  color: var(--primary-color);
+  cursor: pointer;
+  align-self: flex-start;
+}
+
+.SeeCommentsButton:hover {
+  background: var(--gradient-color);
+  color: #fff;
+  transition: 0.2s 0.1s;
+}
+
+.SeeCommentsButton:active {
+  background: var(--primary-color);
+  color: #fff;
+  transition: 0s 0s;
+}
+
+.SeeCommentsButton span {
+  flex: none;
+  order: 0;
+  flex-grow: 0;
```

Comment:
My suggestion here is similar to `margin` part, since `flex: flex-grow flex-shrink flex-basis;` we could just use here `flex: none` which will automatically means `flex-grow: 0`